core/src/autogluon/core/hpo/ray_hpo.py

```diff
@@ -235,7 +235,11 @@ def run(
     )
 
     if not ray.is_initialized():
-        ray.init(log_to_driver=False, **total_resources)
+        ray.init(
+            log_to_driver=False,
+            runtime_env={"env_vars": {"PL_DISABLE_FORK": "1"}},  # https://github.com/ray-project/ray/issues/28197
```

Comment:
This will only be available in `pytorch_lightning>=1.7.4`. See https://pytorch-lightning.readthedocs.io/en/stable/generated/CHANGELOG.html#id6 . We thus need to update the setup.py accordingly.

Reply:
Sure. Let me update it